extensions/wikia/PortableInfoboxBuilder/tests/PortableInfoboxBuilderHelperTest.php

```diff
@@ -34,6 +34,28 @@ public function testActionSubmit( $queryStringValue, $expectedResult ) {
 		$this->assertEquals( $expectedResult, PortableInfoboxBuilderHelper::isSubmitAction( $requestMock ) );
 	}
 
+	/**
+	 * @dataProvider getTitleProvider
+	 */
+	public function testGetTitle( $title, $expected ) {
+		$status = new Status();
+		$this->assertEquals( $expected, PortableInfoboxBuilderHelper::getTitle( $title, $status ) );
+	}
+
+	/**
+	 * @dataProvider createRedirectUrlsProvider
+	 */
+	public function testCreateRedirectUrls( $isGood, $expected ) {
+		$statusMock = $this->getMockBuilder( 'Status' )->setMethods( [ 'isGood' ] )->getMock();
+		$statusMock->expects( $this->any() )->method( 'isGood' )->willReturn( $isGood );
+		$this->mockClass( 'Status', $statusMock );
+		$fullUrlMock = $this->getMockBuilder( 'Title' )->setMethods( [ 'getFullUrl' ] )->getMock();
+		$fullUrlMock->expects( $this->any() )->method( 'getFullUrl' )->willReturn( 'full_url' );
+		$this->mockClass( 'Title', $fullUrlMock );
+
+		$this->assertEquals( $expected, PortableInfoboxBuilderHelper::createRedirectUrls( 'test' ) );
+	}
```

Comment:
not sure if it makes sense to mock everything here. In this way we are testing one `if`. On the other hand, only then we are UNIT testing this function independently.